tests/app/main/views/test_register.py

```diff
@@ -131,7 +131,7 @@ def test_should_return_200_when_email_is_not_gov_uk(
 
 @pytest.mark.parametrize('email_address', (
     'notfound@example.gsa.gov',
-    'example@lsquo.net',
+    pytest.param('example@lsquo.net', marks=pytest.mark.xfail(raises=AssertionError)),
     pytest.param('example@ellipsis.com', marks=pytest.mark.xfail(raises=AssertionError)),
```

Comment:
We should expect this email to raise an assertion error because lsquo.net is not in our assigned domains list